test/routable_component.js

```diff
@@ -34,22 +34,22 @@ class MyController {
 
 /** @test {RoutableComponent} */
 describe('RoutableComponent', function() {
-  lazy('routes', function() { return new RoutableComponentRoutes(this.routing_classes) });
-  lazy('routing_classes', function() { return MyUsefulRouting });
+  lazy('routes', function() { return new RoutableComponentRoutes(this.routingClasses) });
+  lazy('routingClasses', function() { return MyUsefulRouting });
   lazy('components', function() { return {MyEventSourceComponent: new MyEventSourceComponent()} });
-  lazy('controller_classes', function() { return {MyController} });
+  lazy('controllerClasses', function() { return {MyController} });
   lazy('MyEventSourceComponent', function() { return this.subject.components.MyEventSourceComponent });
   /** @test {RoutableComponent#constructor} */
   context('constructed with', function() {
     context('no components', function() {
-      subject(function() { return new RoutableComponent({}, this.routes, this.controller_classes) });
+      subject(function() { return new RoutableComponent({}, this.routes, this.controllerClasses) });
```

Comment:
__[eslint]__
* Error - 'subject' is not defined. (no-undef)
* Warning - Too many nested callbacks (4). Maximum allowed is 3. (max-nested-callbacks)
* Error - Unexpected 'this'. (no-invalid-this)
* Error - Missing semicolon. (semi)

<!-- Comment by [SideCI](https://sideci.com) -->